examples/long_running.py

```diff
@@ -0,0 +1,121 @@
+"""Example specification for a consumption-savings model with health and leisure."""
+
+import jax.numpy as jnp
+
+# ======================================================================================
+# Numerical parameters and constants
+# ======================================================================================
+N_GRID_POINTS = {
+    "states": 100,
+    "choices": 200,
+}
+
+RETIREMENT_AGE = 65
+
+# ======================================================================================
+# Model functions
+# ======================================================================================
+
+
+# --------------------------------------------------------------------------------------
+# Utility function
+# --------------------------------------------------------------------------------------
+def utility(consumption, working, health, sport, delta):
+    return jnp.log(consumption) - (delta - health) * working - sport
+
+
+# --------------------------------------------------------------------------------------
+# Auxiliary variables
+# --------------------------------------------------------------------------------------
+def working(leisure):
+    return 1 - leisure
+
+
+def wage(age):
+    return 1 + 0.1 * age
+
+
+def age(_period):
+    return _period + 18
+
+
+# --------------------------------------------------------------------------------------
+# State transitions
+# --------------------------------------------------------------------------------------
+def next_wealth(wealth, consumption, working, wage, interest_rate):
+    return (1 + interest_rate) * (wealth - consumption) + wage * working
```

Comment:
```suggestion
    return (1 + interest_rate) * (wealth + working * wage - consumption)
```
This would be the usual way to write it down. Also for the constraint below -- I should be able to consume my labor income without having to pay interest on it.

Turn around working / wage: 0/1-indicator upfront eases readability.